grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionState.java

```diff
@@ -110,15 +107,15 @@
     private static final AtomicIntegerFieldUpdater<SessionState> EXPORT_LISTENER_VERSION_UPDATER =
             AtomicIntegerFieldUpdater.newUpdater(SessionState.class, "exportListenerVersion");
 
-    // accumulate the set of used export id's; it is ideal if the api client uses few continuous ranges
-    private final Index usedExportIds = Index.CURRENT_FACTORY.getEmptyIndex();
+    // Usually, export life cycles are managed explicitly with the life cycle of the session state. However, we need
+    // to be able to close non-exports that are not in the map but are otherwise satisfying outstanding gRPC requests.
+    private final LivenessScope nonExportsToClose = new LivenessScope();
```

Comment:
We don't want to do this. Instead, a SimpleReferenceManager of WeakSimpleReferences to Closeables.